docs/getting-started.md

```diff
@@ -69,35 +62,48 @@ The server is accessible via web console at:
     https://192.168.42.147:8443
 ```
 
-Now login to the OpenShift cluster using the server address that was in the output of `minishift start` -
-```console
-$ oc login -u developer -p developer https://192.168.42.147:8443
+Now log into the OpenShift cluster:
+
+```sh
+$ oc login -u developer -p developer
 Login successful.
 
 You have one project on this server: "myproject"
 
 Using project "myproject".
 ```
 
-Make sure you are logged in to the cluster by running `oc whoami` command.
-
 Now we can move on to creating our application using `odo`.
 
-#### Create an application
+### 2. Create an application
 
 An application is an umbrella under which you will build all the components (microservices) of your entire application.
 
-Let's create an application -
+Let's create an application:
 
 ```console
 $ odo app create nodeapp
 Creating application: nodeapp
 Switched to application: nodeapp
 ```
 
-#### Create a component
+### 3. Create a component
 
-Now that you have created an application, now add a component of type _nodejs_ to the application, from the current directory where our code lies.
+First, we'll download the our test application: 
+
+```console
+$ git clone https://github.com/kadel/nodejs-ex
+Cloning into 'nodejs-ex'...
+remote: Counting objects: 568, done.
+remote: Total 568 (delta 0), reused 0 (delta 0), pack-reused 568
+Receiving objects: 100% (568/568), 174.63 KiB | 1.53 MiB/s, done.
+Resolving deltas: 100% (224/224), done.
+
+$ cd nodejs-ex 
+~/nodejs-ex  master
+```
+
+Now that you've created an application, add a component of type _nodejs_ to the application, from the current directory where our code lies:
 
 ```console
 $ odo create nodejs --local=.
```

Comment:
the output shown here is from really old version.
It looks completely different now.